tox.ini

```diff
@@ -310,6 +310,9 @@ deps =
     mako100: mako>=1.0.0,<1.1.0
     mako010: mako>=0.1.0,<1.0.0
     memcached: python-memcached
+    msgpackmin: msgpack<0.6.0
+    msgpack060: msgpack>=0.6.0,<0.6.1
+    msgpacklatest: msgpack>=0.6.0
```

Comment:
I'd just add a flavor that is

`msgpackmin: msgpack==0.5.6`

the latest is the default anyway

so you'd have:
`{py27,py34,py35,py36,py37}-integration{,-msgpackmin}`
above